Cargo.toml

```diff
@@ -31,6 +31,7 @@ tokio = { version = "0.1.21" }
 tokio-fs = { version = "0.1" }
 xdg = "*"
 async-trait = "*"
+async-std = { version = "1.3.0", features = [ "unstable", "std" ] }
```

Comment:
similar question here, any particular reason for picking 1.3 over 1.4 (or even "1"), can't see any breaking changes in their [gh releases](https://github.com/async-rs/async-std/releases)?

Reply:
I think it was just the latest when I started playing with this...